searx/engines/google_images.py

```diff
@@ -61,30 +61,28 @@ def request(query, params):
 
 # get response from search-request
 def response(resp):
+    results = []
+
     dom = html.fromstring(resp.text)
 
-    results = []
-    for element in dom.xpath('//div[@id="search"] //td'):
-        link = element.xpath('./a')[0]
-
-        google_url = urlparse(link.xpath('.//@href')[0])
-        query = parse_qs(google_url.query)
-        source_url = next(iter(query.get('q', [])), None)
-
-        title_parts = element.xpath('./cite//following-sibling::*/text()')
-        title_parts.extend(element.xpath('./cite//following-sibling::text()')[:-1])
-
-        result = {
-            'title': ''.join(title_parts),
-            'content': '',
-            'template': 'images.html',
-            'url': source_url,
-            'img_src': source_url,
-            'thumbnail_src': next(iter(link.xpath('.//img //@src')), None)
-        }
-
-        if not source_url or not result['thumbnail_src']:
+    # parse results
+    for result in dom.xpath('//div[contains(@class, "rg_meta")]/text()'):
+
+        try:
+            metadata = loads(result)
+            format = metadata['ity'] + ' ' + str(metadata['ow']) + 'x' + str(metadata['oh'])
```

Comment:
you can use string formatter `"{0} {1}x{2}".format(metadata['ity'], metadata['ow'], metadata['oh'])